crates/node/builder/src/setup.rs

```diff
@@ -39,7 +40,7 @@ pub fn build_networked_pipeline<DB, Client, Executor>(
 ) -> eyre::Result<Pipeline<DB>>
 where
     DB: Database + Unpin + Clone + 'static,
-    Client: HeadersClient + BodiesClient + Clone + 'static,
+    Client: BlockClient + Clone + 'static,
```

Comment:
```suggestion
    Client: BlockClient + 'static,
```
`Clone` is already supertrait of `BlockClient`

Reply:
Resolved in https://github.com/paradigmxyz/reth/commit/a00d119bd93e09d554a167f40a5d3aaeb1a86a6c